docs/README.md

```diff
@@ -45,9 +45,20 @@ Check an example of a Product Details Page built using Flex Layout with the `pro
   },
 ```
 
-| Prop name                  | Type     | Description                                                                                                                                                             |
-| -------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). |
+| Prop name | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). | `0` |
+| `size` | `NumericSize`| Preset values `font-size` and `padding` to the component | `'small'` |
```

Comment:
How about breaking this up into two props, one for the `padding` and one for the `font-size`? 

Also, why not make their values of type `number` and treat them as values for Tachyons tokens, the same way it is done in our `flex-layout`?
Example: https://github.com/vtex-apps/flex-layout/blob/master/react/Row.tsx#L53

Reply:
I don't think these enum values of `"small"`, `"medium"` and `"large"` are much intuitive. I mean, they are intuitive, but their values are arbitrary for the user.

Reply:
This is something from styleguide since we're using it I'm just using the same semantic of it. This is just a facilitator to change these values if you want something very specific you can always rely on the CSS handles.

Reply:
Oooh, I see :(
That's fine then, but I think we should avoid this in other cases when we have full control over the styling.